packages/pv-create-component/lib/templates/mdTemplate.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function({name, componentName, type}) {
+  return `---
+name: ${name}
+category: ${type}s
+---
+
+@TODO: Descriptive Text goes here.
+
+\`\`\`${componentName}:demo/${componentName}--demo.html
```

Comment:
should this default to a demo file?

Reply:
i find it more consistent if all components have a demo file rather than only some